doc/user/content/serve-results/snowflake.md

```diff
@@ -0,0 +1,255 @@
+---
+title: "Snowflake"
+description: "How to export results from Materialize to Snowflake."
+menu:
+  main:
+    parent: sink
+    name: "Snowflake"
+    weight: 45
+---
+
+[//]: # "TODO(morsapaes) For Kafka users, it's possible to sink data to
+Snowflake continuously using the Snowflake connector for Kafka. We should also
+document that approach."
+
+{{< private-preview />}}
+
+This guide walks you through the steps required to bulk-export results from
+Materialize to Snowflake using Amazon s3 as the intermediate object store.
+
+## Before you begin
+
+- Ensure you have access to an AWS account, and permissions to create and manage
+  IAM policies and roles. If you're not an account administrator, you will need
+  support from one!
+
+- Ensure you have access to a Snowflake account, and are able to connect as a
+  user with either the [`ACCOUNTADMIN` role](https://docs.snowflake.com/en/user-guide/security-access-control-considerations#using-the-accountadmin-role),
+  or a role with the [global `CREATE INTEGRATION` privilege](https://docs.snowflake.com/en/user-guide/security-access-control-privileges#global-privileges-account-level-privileges).
+
+## Step 1. Set up bulk exports to Amazon s3
```

Comment:
Capitalize S3. Just do a global search replace. I‘ll stop  commenting every one :-)

Reply:
Can't search and replace globally because it must be lower-cased in certain places, but just did another pass — thank you!